jameda_config.js

```diff
@@ -1,7 +1,19 @@
 /* eslint-disable no-unused-vars, no-var */
 
-var isCfgEnabled = parseURLParams(window.location, true, 'search').cfg || false;
+const urlParams = parseURLParams(window.location, true, 'search');
 
+/**
+ * Set a global configuration object
+ *
+ * @type {{isCfgEnabled: boolean, peerBrowser: string}}
+ */
+window.jameda = {
+    isCfgEnabled: false,
```

Comment:
Please assign right away to `isCfgEnabled: urlParams.isCfgEnabled || false;`, so we dont need the below assignments values.